constraint/pkg/client/client.go

```diff
@@ -539,7 +578,7 @@ func (c *Client) getConstraintNoLock(ctx context.Context, constraint *unstructur
 		return nil, err
 	}
 
-	cstr, ok := c.constraints[subPath]
+	cstr, ok := c.constraints[constraint.GroupVersionKind().GroupKind()][subPath]
```

Comment:
Yay to Go supporting lookup on nil maps! :)

Reply:
Oh yes! Assigning however... :p